routers/carsRouter.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const knex = require('knex');
+const db = require('../data/dbConfig');
+const { validateId } = require('../middleware/validation');
+
+const router = express.Router();
+
+router.get("/", async (req, res, next) => {
+  try {
+    res.status(200).json(await db("cars").select())
+  } catch (err) {
+    res.status(500).json({
+      err: err,
+      errorMessage: 'The cars information could not be retrieved.'
+    })
+    next(err)
```

Comment:
`next(err)` automatically passes along the req object to the error handler middleware but I don't see one created.  Furthermore, `res.json()` acts similar to a return in JS so the `next(err)` will never get called